protocol/forwardprotocol/message.go

```diff
@@ -54,3 +62,35 @@ func init() {
 	unusedStruct(EventEntry{}._msgpack)
 	unusedStruct(TransportOption{}._msgpack)
 }
+
+// ResolvePath attempts to fetch field by the given path
+//
+// Supports nested maps, e.g. path=http/statusCode
+func (e *EventEntry) ResolvePath(path string) (interface{}, error) {
```

Comment:
Should we check path before executing doResolvePath?
If we get an invalid path parameter - there is no need to try to resolve it.

Reply:
But what constitutes as invalid? There is no restriction on the contents of map key (even empty string), except we have to pick '/' or some other character as the path separator, or maybe just take a list of strings?